apps/backend/src/datasources/stfc/UOWSClient.ts

```diff
@@ -0,0 +1,14 @@
+import dotenv from 'dotenv';
+
+import { UOWS } from '../../generated';
+
+dotenv.config({ path: process.env.CONFIG_PATH });
+
+export function createUOWSClient() {
+  return new UOWS({
+    BASE: process.env.EXTERNAL_UOWS_API_URL,
+    HEADERS: {
+      Authorization: `Api-key ${process.env.API_KEY}`,
+    },
```

Comment:
I know the old key will still be working but can you have a chat with users about updating to new keys.